docs/source/tutorial/typesfuns.rst

```diff
@@ -1262,14 +1262,28 @@ These let bindings can be annotated with a type:
    mirror xs = let xs' : List a = reverse xs in
                    xs ++ xs'
 
-We can also use the symbol ``:=`` instead of ``=`` to, among other things,
-avoid ambiguities with propositional equality:
+Since ``=`` can either indicate a type of equality (``===`` or ``~=~``) or a
+definition, some expressions can be ambiguous. Here is an example:
+
+.. code-block:: idris
+
+   -- Diag : a -> Type
+   -- Diag v = let ty : Type = v = v in ty
+   --                        ^
+   --                        |
+   -- Doesnt compile! because ambiguous here
+
+We can also use the symbol ``:=`` instead of ``=`` in this context to, among
+other things, avoid these ambiguities with propositional equality:
 
 .. code-block:: idris
 
    Diag : a -> Type
    Diag v = let ty : Type := v = v in ty
 
+The code above can be read as "``ty`` has type ``Type`` and it's value
```

Comment:
```suggestion
The code above can be read as "``ty`` has type ``Type`` and its value
```

Reply:
Done and also fixed this error in another page :+1: